packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -1,4 +1,36 @@
 # terra-avatar Upgrade Guide
+## Changes from version 2 to version 3
+
```

Comment:
@ryanthemanuel, do we want to add a 'Reason for change' section? If so, what should it say?

Reply:
+1.  We should add that in the upgrade guide as well as the change log.